conductor-client/src/main/java/transforms/GeocoderTransform.java

```diff
@@ -19,7 +19,7 @@
 import java.util.Optional;
 
 public class GeocoderTransform extends Transformation<Object> {
-    protected static final String       NOMINATIM_SERVICE_URL = "https://osm.openlattice.com/nominatim/";
+    protected static final String       NOMINATIM_SERVICE_URL = "https://osm.openlattice.com";
```

Comment:
I think we need it to be: `https://osm.openlattice.com/` so the requests end up like:
`https://osm.openlattice.com/search?q=94118` (plus the other arguments)

Everything was previously of the form `https://osm.openlattice.com/nominatim/search?q=94118`

Reply:
sounds good, I was wondering if we needed the slash but wasn't sure.